test/steps/utils.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Looks like someone forget to run the unit tests. :)

Reply:
Well, it was me but I thought CI would catch this... So the unit tests are currently not run when opening a PR? I just run BDD tests as I know these are not executed automatically.

Reply:
unit tests are run in CircleCI

Reply:
but what @ricardozanini is saying there is that before comitting, we should run the unit tests to have the header automatically added the fhe files

Reply:
I see, thanks!